core/src/avm2/globals/flash/events/MouseEvent.as

```diff
@@ -76,6 +76,17 @@ package flash.events
             return this.formatToString("MouseEvent","type","bubbles","cancelable","eventPhase","localX","localY","stageX","stageY","relatedObject","ctrlKey","altKey","shiftKey","buttonDown","delta");
         }
 
+        public function get isRelatedObjectInaccessible():Boolean {
+            return _isRelatedObjectInaccessible;
+        }
+
+        public function set isRelatedObjectInaccessible(value:Boolean):void {
+            // Technically this sets exactly what it should.
+            // However, it is a stub since it affects nothing.
+            stub_setter("flash.events.MouseEvent", "isRelatedObjectInaccessible");
```

Comment:
Doesn't matter if it affects nothing, that's what it's meant to do, it's not a stub

Reply:
Ok. I assume the same applies to the properties I added in #14037 too then so I should remove the stub_setters there too, correct?

Reply:
Yep, thank you!